### DISCUSSION TITLE
Discrepancy between seasons in Jellyseer and Sonarr

### DISCUSSION THREAD

**Discussion by nao-hoshino** [seerr-team/seerr#944] [Q&A] [answered]
Hi all,

Since this isn't a bug, it didn't feel right to make an issue report and thought that I would post it here instead.

There is a specific show/anime in Jellyseer which isn't getting requested properly.

The show name is 'Princess Tutu' and Jellyseer states that it has 38 episodes in 1 season.

However, in Sonarr the same show has these 38 episodes spread between 2 seasons.

As a result Jellyseer only requests season 1, while season 2 has to be downloaded manually via Sonarr.

Any thoughts on whether to create an issue for this, and whether it should go here or in the Overseer issue tracker?

**Comment by gauthier-th** [comment, marked as answer]
We use TMDB to get data from tv shows. Sonarr uses TVDB, so that's why there might be differences between Jellyseerr/Sonarr.
You can look at #899 which is trying to implement TVDB in Jellyseerr.

**Comment by nao-hoshino** [comment]
Ah ok that makes sense - thank you!
